task/coverity-availability-check/0.1/coverity-availability-check.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: sast-coverity-availability-check
```

Comment:
@jperezdealgaba I think the name should rather be `coverity-availability-check`.

Reply:
Sure.  Fixed!